docs/releases/index.md

```diff
@@ -39,11 +39,11 @@ Release highlights:
      3PAR OS 3.3.1, 3.3.2
    </td>
  </tr>
- <!-- tr>
+ <tr>
    <th>Blogs</th>
    <td>
-    <a href="https://FIXME">HPE Alletra Observability for DevOps and Cloud Native with Prometheus</a><br />
-    <a href="https://FIXME">Get started with Prometheus and HPE Alletra</a> (tutorial) <br />
+    Release blog on <a href="https://community.hpe.com/t5/Around-the-Storage-Block/HPE-CSI-Driver-for-Kubernetes-enhancements-with-monitoring-and/ba-p/7158137">Around The Storage Block</a><br />
+    Tutorial using Docker on <a href="https://developer.hpe.com/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter/">HPE Developer Community</a>
```

Comment:
We are in a table.  Should we use separate cells for each item?  (I haven't seen this rendered, so I'm guessing you already decided it is better this way.)

Reply:
I force line-breaks within the cell. 
<img width="788" alt="image" src="https://user-images.githubusercontent.com/7026024/151264774-e07521d0-0cdb-45c6-9476-969216009f89.png">